src/binder/assemblybinder.cpp

```diff
@@ -854,26 +854,22 @@ namespace BINDER_SPACE
                                    &bindResult)) == HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND))
                 )
             {
-                if ((fInspectionOnly && !bindResult.HaveResult()) ||
-                    (bindResult.GetRetargetedAssemblyName() == NULL))
+                IF_FAIL_GO(CreateImageAssembly(pIMetaDataAssemblyImport,
+                                                PeKind,
+                                                pPEImage,
+                                                NULL,
+                                                fInspectionOnly,
+                                                &bindResult));
+                if (fInspectionOnly)
```

Comment:
`fInspectionOnly` should be always false in CoreCLR. I think this flag can be deleted.